yamls/baselines/colbert/bert-base-uncased.yaml

```diff
@@ -0,0 +1,31 @@
+# This YAML is built to work with the `sequence_classification.py` starter script!
+#
+#   Follow the instructions in that script to modify the `build_my_dataloader` function
+#   and fine-tune a BERT model on your own dataset!
+#
+#
+#   Note that some of the fields in this template haven't been filled in yet.
+#   Please resolve any empty fields before launching!
+
+# Run Name
+run_name: finetune-hf-bert
+
+seed: 42
+model_name_or_path: bert-base-uncased
+train_dataset_id: bclavie/msmarco-500k-triplets-colbert-format
+n_gpu: 1
+tmp_dir: ./tmp_colbert
+debug: false
+
+train_params:
+  lr: 3e-5
```

Comment:
Original ColBERT v1 paper lr is 3e-6 (v2 is 1e-5), you put this value because of empirical results?

Reply:
> you put this value because of empirical results
Yes and also because overall the results are that bigger models struggle to learn with the tiny LR -- even RoBERTa-large does poorly with 3e-6. We can turn it down to 1e-5 however to be closer to the paper, we should be enough for all ablations at least.

Reply:
No I do not mind using another (empirically better) lr, especially if we are comparing our own models.
Was being nitpick since I did not know if we wanted to compare to the paper results.